src/main/java/no/nav/pto/veilarbportefolje/arenapakafka/ytelser/YtelsesService.java

```diff
@@ -63,18 +63,6 @@ public void behandleKafkaRecord(ConsumerRecord<String, YtelsesDTO> kafkaMelding,
         arenaHendelseRepository.upsertYtelsesHendelse(innhold.getVedtakId(), innhold.getHendelseId());
     }
 
-    public void behandleKafkaRecordV2(ConsumerRecord<String, YtelsesDTO> kafkaMelding, TypeKafkaYtelse ytelse) {
```

Comment:
why to remove this :)

Reply:
you were testing something in prod? :)

Reply:
It is not really a V2 method it is sort of a read message twice method.
The V1 method updates both postgres and oracle and has a check so no message is read twice.
I just needed this method when I migrated old data